ggml/src/ggml-sycl/dpct/helper.hpp

```diff
@@ -2624,6 +2687,7 @@ namespace dpct
                 beta, c, ldc, stride_c, batch_size);
             break;
         }
+#endif
```

Comment:
Same as above, the `#ifdef __INTEL_MKL__` block should only cover the first two type-combinations that are not supported by non intel targets.